### ISSUE TITLE
Align a select and an input on bottom - issue #611- still not working

### ISSUE THREAD

**Issue by jstorry** [Dogfalo/materialize#839]
Doesn't look like  https://github.com/Dogfalo/materialize/commit/ae38b2907af92f310a3df08c9542ae48e6468b7a quite fixed the issue. The line under the select appears to be off by one pixel when the width of the screen reaches a certain size, and when you add a label to the select it gets worse.

Here's an example using 0.95.3
http://jsfiddle.net/0nn8jwrs/

**Comment by djhi** [comment]
I solved this prb with this css:
```
.select-wrapper {
  margin: 0 0 15px 0;
}
```

and using this html:
```
<div class="col s12 m4 input-field">
    <label for="select-id" class="active">label</label>
    <select id="select-id">
        <option value="" selected> Select one</option>
        <option value="value1">Value 1</option>
        <option value="value2">Value 2</option>
    </select>
</div>
```

Hope it helps !

**Comment by acburst** [comment]
Fixed in 94eec79d7a88cc5513333624f4d8b4214b3a123a.

There are some structure changes. The select wrapper must now be in a ```.input-field``` div, and the label must come after the select wrapper

```
        <div class="input-field col s4">
          <select>
            <option value="" disabled selected>Choose your option</option>
            <option value="1">Option 1</option>
            <option value="2">Option 2</option>
            <option value="3">Option 3</option>
          </select>
          <label>Materialize Select</label>
        </div>
```
